common/timedcache/timedcache.go

```diff
@@ -51,22 +59,26 @@ func (tc *TimedCache) removeExpired() {
 
 // Purge is used to completely clear the cache.
 func (tc *TimedCache) Purge() {
```

Comment:
just making sure that we purge, the futureHeadersCache

Reply:
I don't follow